android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/TimePickerDialogFragment.kt

```diff
@@ -125,6 +126,17 @@ internal class TimePickerDialogFragment :
         onClick(dialog, BUTTON_NEGATIVE)
     }
 
+    override fun onStart() {
+        super.onStart()
+
+        val alertDialog = dialog
+        if (alertDialog is AlertDialog) {
+            // We want to call the extension function after the show() call on the dialog,
+            // and the DialogFragment does that call during onStart().
+            alertDialog.withCenterAlignedButtons()
```

Comment:
Maybe we should add this warning on of the extension documentation as a warning for future consumers.